sdk/eidolon_ai_sdk/util/filter_json.py

```diff
@@ -0,0 +1,69 @@
+from jsonpath_ng.ext import parse
+
+
+def filter_and_reconstruct_json(original_json, result_filters):
```

Comment:
How is this used? Is it so that we don't pass too large of an object back to the llm?

Not that I don't trust your json parsing, but can we parse the whole thing and then iterate the python object instead? It is less efficient for sure, but I don't think that will kill us. If the json parser is implemented in cython it would also be much more efficient.

Reply:
I'm using a library to remove the bits I don't need. The user specifies that. The original json is there so I can reconstruct it. So I think I'm doing what you are asking for.